spec/unit/helper_spec.rb

```diff
@@ -0,0 +1,164 @@
+require_relative 'spec_helper'
+include CernerSplunk::PathHelpers, CernerSplunk::RestartHelpers, CernerSplunk::ResourceHelpers
```

Comment:
This might just be me, but I would prefer separate test files for every helper to isolate everything.